lib/test-ast.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const fs = require('fs');
+const AST = require('recast');
+
+/* Lives throughout one test run as `watch` mode is not supported */
+const fileAstCache = {};
+
+/**
+ * Retrieve AST for a test starting at line number with a given title
+ *
+ * @param test      Start line number and title of test function
```

Comment:
Why not use object destructuring and make this self documenting?

(I have a severe dislike of these JSDoc comments, sorry.)

Reply:
I'm totally with you on this, and that's what I had at first. Then the linter swung by to whine about destructuring anything but the last argument.

But if you're willing to change the linter settings, I'll be happy to revert. Still found this better than nothing as "test" can mean different things.

Reply:
You might be right. I'd prefer keeping them as separate parameters, as I feel the "test" and the file that it's enclosed in are not the same concept. Do you always use `disable-line` or just disable the specific rule? I prefer the latter as it describes intent and still captures other potential errors if the code were to change in other ways.

Reply:
> I'd prefer keeping them as separate parameters, as I feel the "test" and the file that it's enclosed in are not the same concept.

Yea it really doesn't matter for such an internal function.



> Do you always use `disable-line` or just disable the specific rule? I prefer the latter as it describes intent and still captures other potential errors if the code were to change in other ways.

XO won't allow you to disable the line 😉